app/views/events/_event.html.erb

```diff
@@ -16,5 +16,14 @@
           </div>
         <% end %>
     <% end %>
+    <% if event.sponsors.present? %>
+      <h3>Brought to you by our sponsors: </h3>
+      <% event.sponsors.each do |sponsor| %>
+        <h4><%= link_to sponsor.name, organization_path(sponsor) %></h4>
+        <% if sponsor.logo %>
+          <%= image_tag sponsor.logo.thumbnail.url %>
```

Comment:
There's a `cl_image_tag` helper provided with the gem (very useful for "on-the-fly" formats)

```
<%= cl_image_tag sponsor.logo.thumbnail %>
```